go.mod

```diff
@@ -18,9 +18,9 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.11.0
-	github.com/prometheus/common v0.26.0
+	github.com/prometheus/common v0.29.0
```

Comment:
Why has this changed?  Doesn't seem to be referenced in `weaveworks/common`.
Similarly all the `golang.org/x` changes.

Reply:
Update: weaveworks/common/go.mod doesn't list github.com/prometheus/common as a dependency, but if I upgrade it to Go 1.17, github.com/prometheus/common gets listed among indirect dependencies at v0.29.0. I guess that's why `go mod tidy` upgrades it for dskit, due to an indirect dependency of weaveworks/common.

Reply:
@bboreham Through bisecting, I've found that commit [953ac9fb41437fee0bffcb364333ba624a35e043](https://github.com/weaveworks/common/commit/953ac9fb41437fee0bffcb364333ba624a35e043) introduces the indirect dependency on github.com/prometheus/common@v0.29.0. Before that, weaveworks/common depends on v0.9.1 of github.com/prometheus/common.

Reply:
Thanks.  I see `TLSConfig` used in `weaveworks/common` depends on a type `Secret` in `prometheus/common`.

Reply:
Confirmed with Bryan offline that this change is OK.